packages/amplify_api/pubspec.yaml

```diff
@@ -4,7 +4,7 @@ version: 0.1.4
 homepage: https://github.com/aws-amplify/amplify-flutter/tree/master/packages/amplify_api
 
 environment:
-  sdk: ">=2.7.0 <3.0.0"
+  sdk: ">=2.12.0 <3.0.0"
   flutter: ">=1.20.0"
```

Comment:
Maybe we could switch to the
```yaml
flutter: ^1.20.0
```
for consistency?

Reply:
sounds good, done